java/runtime-common/src/main/java/org/ray/spi/LocalSchedulerProxy.java

```diff
@@ -131,4 +135,33 @@ public void reconstructObject(UniqueID objectId) {
   public void notifyUnblocked() {
     scheduler.notifyUnblocked();
   }
+
+  private static byte[][] getIdBytes(List<UniqueID> objectIds) {
+    int size = objectIds.size();
+    byte[][] ids = new byte[size][];
+    for (int i = 0; i < size; i++) {
+      ids[i] = objectIds.get(i).getBytes();
+    }
+    return ids;
+  }
+
+  public <T> WaitResult<T> wait(RayList<T> waitfor, int numReturns, int timeout) {
```

Comment:
ObjectStoreProxy already have the wait functions, which from the JNI of plasma and open to the users by Ray.wait, so i am not sure the relationship between the wait function here in LocalSchedulerProxy and from JNI of ls, thanks.

Reply:
And do not found any usage of this function.

Reply:
Got it, it will be used for raylet.

Reply:
[This](https://github.com/ant-tech-alliance/ray/blob/java_raylet_2/java/runtime-common/src/main/java/org/ray/spi/ObjectStoreProxy.java#L103) invokes it.